### PR TITLE
Upgrade to bpmn-js 6.2 and provide deviceScaleFactor

### PR DESCRIPTION
**Pull request by huksley**
## What have been done?

- [x] Upgrade to 6.2.0 bpmn-js
- [x] Ran npm audit fix
- [x] Add --scale N (default 1) to produce better quality raster images (better DPI)

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -98,14 +102,17 @@ const footer = cli.flags.footer;
 
 const title = cli.flags.title === false ? false : cli.flags.title;
 
+const scale = cli.flags.scale !== undefined ? cli.flags.scale : 1;
+
 const [ width, height ] = cli.flags.minDimensions.split('x').map(function(d) {
   return parseInt(d, 10);
 });
-
+console.info(cli.flags)
```
Comment (nikku) [inline cli.js]: Could you remove the console.info log here?

File: package.json [hunk 2]
```diff
@@ -25,10 +25,10 @@
     "png",
     "svg"
   ],
-  "author": "Nico Rehwaldt <https://github.com/nikku>",
+  "author": "Nico Rehwaldt <https://github.com/nikku>, Ruslan Gainutdinov <https://github.com/huksley>",
```
Comment (nikku) [inline package.json]: Please remove yourself from the authors. You may add yourself to the list of contributors, cf. [npm documentation](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) if you'd like to.

File: cli.js (hunk 1 above)
Comment (huksley) [reply cli.js]: Removed

File: package.json (hunk 2 above)
Comment (huksley) [reply package.json]: Removed

**Comment by nikku** [comment]
Squashed and merged via https://github.com/bpmn-io/bpmn-to-image/commit/a2015f3817cbe818a4e81eef609caf0dfbccc6eb.

Released as `v0.5.1`. 

Thanks for your contribution.

**Pull request closed without merging**